pkg/api/handlers.go

```diff
@@ -131,6 +132,22 @@ func configHandler(cfg interface{}) http.HandlerFunc {
 	}
 }
 
+func overridesHandler(runtimeCfgManager *runtimeconfig.Manager) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Please note that this has now been done in https://github.com/cortexproject/cortex/pull/3645 too. To avoid duplicate work, it may be good idea to rebase on top of recent master, once https://github.com/cortexproject/cortex/pull/3645 is merged.

Reply:
Great, thank you for pointing me to the new change. I'll wait until that PR is merged to make the rebase

Reply:
I just merged #3645, so you can rebase `master` and just use that helper.